diagnosis/getkuberneteslogs.sh

```diff
@@ -40,8 +150,13 @@ function printUsage
     echo "  -i, --identity-file             RSA private key tied to the public key used to create the Kubernetes cluster (usually named 'id_rsa')"
     echo "  -m, --master-host               A master node's public IP or FQDN (host name starts with 'k8s-master-')"
     echo "  -d, --vmd-host                  The DVM's public IP or FQDN (host name starts with 'vmd-')"
+    echo "  --spn-client-id                 Service Principle client Id used to create the Kubernetes cluster"
+    echo "  --spn-client-secret             Service Principle client secret used to create the Kubernetes cluster"
+    echo "  -t, --tenant-id                 Tenant id"
```

Comment:
Inconsistency, previous line uses `Id`. I prefer `ID`, the azure portal uses that.